tests/construction/dj_query_test.py

```diff
@@ -74,17 +74,12 @@ async def test_build_dj_metric_query(request):
         AND basic_DOT_num_users_us.b_DOT_secret_number = basic_DOT_source_DOT_users.secret_number
     LEFT OUTER JOIN (
       SELECT
-        basic_DOT_source_DOT_users.age,
         basic_DOT_source_DOT_users.country,
-        basic_DOT_source_DOT_users.full_name,
-        basic_DOT_source_DOT_users.gender,
-        basic_DOT_source_DOT_users.id,
-        basic_DOT_source_DOT_users.preferred_language,
-        basic_DOT_source_DOT_users.secret_number
+        basic_DOT_source_DOT_users.id
```

Comment:
A nice side-effect here is cleaner generated SQL 😄 (besides making the query engine's job easier, of course)

Reply:
Haha, yes I can actually read the queries now 😂